setup.cfg

```diff
@@ -0,0 +1,42 @@
+[metadata]
+name = qutip
+url = http://qutip.org
+description = QuTiP: The Quantum Toolbox in Python
+long_description = file: README.md
+long_description_content_type = text/markdown; variant=GFM
+keywords = quantum, physics, dynamics
+license = BSD 3-Clause License
+license_file = LICENSE.txt
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Topic :: Scientific/Engineering
+    Operating System :: MacOS
+    Operating System :: POSIX
+    Operating System :: Unix
+    Operating System :: Microsoft :: Windows
+author = Alexander Pitchford, Paul D. Nation, Robert J. Johansson, Chris Granade, Arne Grimsmo, Nathan Shammah, Shahnawaz Ahmed, Neill Lambert, Eric Giguere, Boxi Li
+author_email = qutip-admin@googlegroups.com
+platforms = Linux, Mac OSX, Unix, Windows
+
+[options]
+packages = find:
+include_package_data = True
+zip_safe = False
+install_requires =
+    numpy>=1.12
+    scipy>=1.0
+setup_requires =
+    numpy>=1.16.6,<1.20
+    scipy>=1.0
+    cython>=0.29.20
```

Comment:
@BoxiLi: the different install vs setup limits are here

Reply:
@Ericgig this is a different PR to the merge, but it's clearer to see the difference in requirements in the new packaging methods (note that I'll likely be rebasing/redoing this PR if #1453 is merged anyway, or backporting it to master)

Reply:
(also note the numpy `install_requires` version in this PR should be bumped up to 1.16.6 - whoops!)